neqo-transport/src/send_stream.rs

```diff
@@ -933,4 +939,67 @@ mod tests {
         let res = tx.next_bytes(TxMode::Normal);
         assert_eq!(res, None);
     }
+
+    #[test]
+    fn send_stream_avail() {
+        let flow_mgr = Rc::new(RefCell::new(FlowMgr::default()));
+        flow_mgr.borrow_mut().conn_increase_max_credit(2);
+        let conn_events = ConnectionEvents::default();
+
+        let mut s = SendStream::new(4.into(), 0, flow_mgr.clone(), conn_events.clone());
```

Comment:
You should make a new stream with available flow control from the start to see that you generate the event out of `::new()`

Reply:
will do. Found a bug as a result, hah.